packages/webdriver/webdriver.d.ts

```diff
@@ -181,6 +181,17 @@ declare namespace WebDriver {
         level: FirefoxLogLevels
     }
 
+    export interface GeckodriverCapabilities {
```

Comment:
Already exists in the types.ts file, like I said in gitter. We should find a standard for ourself to prevent these kind of mistakes :)

Reply:
yes, for now we have to maintain two sources of truth and once we have rewritten everything we can clean everything up.